<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>160</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>512</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>512</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>491</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>454</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="dataPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>32</y>
      <width>491</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>/home/ringo/.local/share/openxcom/data/GEODATA/SCANG.DAT</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="exportPath">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>50</y>
      <width>491</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>/tmp/tiles</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="exportBtn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QPushButton" name="palettesPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>/home/ringo/.local/share/openxcom/data/GEODATA/PALETTES.DAT</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
